src/Content.tsx

```diff
@@ -72,11 +60,7 @@ function injectButtons(): void {
               //   <UpvoteDownvoteButton professorName={inst.innerText} />
               // );
 
-              createRoot(rateMyProfessorRoot).render(
-                <RateMyProfessorButton professorName={inst.innerText} />
-              );
-
-              createRoot(professorPopupRoot).render(
+              createRoot(parentElem).render(
                 <ProfessorPopup professorName={inst.innerText} />
```

Comment:
I can't find exactly where the culprit is, but courses with "To be Announced" have the text deleted in the professorpopup button

With extension:
![image](https://user-images.githubusercontent.com/24869207/228998079-47a8e283-f410-448c-826e-3a73ad7e3024.png)

Without:
![image](https://user-images.githubusercontent.com/24869207/228998116-f1564b9d-353c-40ab-a9eb-a7f47313efef.png)